documentation/sphinx/source/release-notes/release-notes-730.rst

```diff
@@ -0,0 +1,73 @@
+#############
+Release Notes
+#############
+
+7.3.0
+=====
+
+Fixes
+-----
+* Added back samples for (non)empty peeks stats.
+`(PR #9072) <https://github.com/apple/foundationdb/pull/9072>`
+* Fixed a consistency scan infinite looping without progress bug when a
+storage server is removed.
+`(PR #9154) <https://github.com/apple/foundationdb/pull/9154>`
```

Comment:
These should be in one line. Otherwise, the rendering of this part is not bullet, but a paragraph.